.github/workflows/test.yaml

```
@@ -0,0 +1,26 @@
+on:
+  pull_request:
+    branches:
+      - main
+      - chungjung/issue40
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name : Checkout
```

yamllint (1.29.0-1)

[warning] too many spaces before colon

(colons)